src/main/java/bitflow4j/steps/database/WriteHistogramToElasticsearch.java

```diff
@@ -0,0 +1,89 @@
+package bitflow4j.steps.database;
+
+import bitflow4j.Sample;
+import bitflow4j.steps.BatchHandler;
+import org.apache.http.HttpHost;
+import org.elasticsearch.action.DocWriteResponse;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.index.IndexResponse;
+import org.elasticsearch.action.support.replication.ReplicationResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentFactory;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class WriteHistogramToElasticsearch implements BatchHandler {
```

Comment:
Please add a toString() method with something like: "Output to Elasticsearch at host:port, host:port in index 'index name' with key=template"